requirements.txt

```diff
@@ -70,3 +70,6 @@ pytest-asyncio==0.20.3
 pytest-cov==4.0.0
 pytest-timeout==2.1.0
 pytest-xdist==3.1.0
+types-redis
```

Comment:
We only add these to please mypy, that is not the way to go (see other comments).